src/utils/interfaces.ts

```diff
@@ -0,0 +1,16 @@
+export interface IShopping {
```

Comment:
It's a bad name for the interface. Better call it something like IPurchase or something